tools/RAiDER/cli/raider.py

```diff
@@ -17,6 +18,8 @@
 from RAiDER.cli.validators import DateListAction, date_type
 from RAiDER.models.allowed import ALLOWED_MODELS
 from RAiDER.utilFcns import get_dt
+from RAiDER.aria.prepFromGUNW import main as GUNW_prep
+from RAiDER.aria.calcGUNW import tropo_gunw_slc as GUNW_calc
```

Comment:
I moved this outside the function Brett so monkey patching would work.

We have `calcGUNW` and `GUNW_calc` and ... - would be in our interest to have better names for these @bbuzz31